### ISSUE TITLE
Motion Prediction using Trajectory Sets and Self-Driving Domain Knowledge. (arXiv:2006.04767v2 [cs.LG] UPDATED)

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#2799]
Motion Prediction using Trajectory Sets and Self-Driving Domain Knowledge. (arXiv:2006.04767v2 [cs.LG] UPDATED)<br>
https://ift.tt/38HW9lI<br>
<p>Predicting the future motion of vehicles has been studied using various techniques, including stochastic policies, generative models, and regression. Recent work has shown that classification over a trajectory set, which approximates possible motions, achieves state-of-the-art performance and avoids issues like mode collapse. However, map information and the physical relationships between nearby trajectories is not fully exploited in this formulation. We build on classification-based approaches to motion prediction by adding an auxiliary loss that penalizes off-road predictions. This auxiliary loss can easily be pretrained using only map information (e.g., off-road area), which significantly improves performance on small datasets. We also investigate weighted cross-entropy losses to capture spatial-temporal relationships among trajectories. Our final contribution is a detailed comparison of classification and ordinal regression on two public self-driving datasets.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
